packages/valory/skills/market_manager_abci/bets.py

```diff
@@ -58,6 +68,32 @@ class Bet:
     status: BetStatus = BetStatus.UNPROCESSED
     blacklist_expiration: float = -1
 
+    @classmethod
+    def from_gamma_subgraph(cls, **kwargs: Any) -> "Bet":
+        """Initialize a bet's instance from gamma subgraph's attributes."""
+        outcomes = eval(kwargs["outcomes"])
+        outcome_token_amounts = [0] * len(outcomes)
```

Comment:
We need to figure out how to calculate the `outcome_token_amounts`.

Reply:
The subgraph returns zero for all the markets' amounts because Polymarket is not adding liquidity to the FPMMs.